pkgs/tools/system/uefitool/new-engine.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  qtbase,
+  cmake,
+  wrapQtAppsHook,
+  zip,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "uefitool";
+  version = "A71";
+
+  src = fetchFromGitHub {
+    sha256 = "sha256-NRlrKm5+eED6oyvFRSEhn0EUbMsPJtuFAyv3vgY/IUI=";
+    owner = "LongSoft";
+    repo = pname;
+    rev = version;
```

```suggestion
    repo = "uefitool";
    tag = version;
```
the repo shouldn't change when overriding the `pname` and a few months ago the `tag` attr was added to the gh fetcher (see #355973 ).